docs/modules/reference/pages/provisioning/detectors/LoopDetector.adoc

```diff
@@ -0,0 +1,55 @@
+= Loop Detector
+
+The Loop Detector gives {page-component-title} users the ability to force a service onto an interface during service detection.
+This can be used to ensure a specific service is added to all nodes, without performing a check against the node.
+The primary use case for this type of detector is when using xref:reference:service-assurance/monitors/PassiveStatusMonitor.adoc[passive service monitoring].
+
+First, add a detector for the new service to the desired foreign-source definition.
+Use the LOOP class:
+
+Add parameters:
+
+* ipMatch : 169.254.*.*
```

Comment:
There's an extra period appearing as bold in the docs. But it doesn't make sense to user literals here, as it would wreck the example. Just an observation.

Reply:
```suggestion
* ipMatch : `169.254.*.*`
```